content/news/draft-bevy-0.14/index.md

```diff
@@ -22,7 +22,7 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <!-- more -->
 
-{{ release_notes(version="0.14") }}
+{{ release_notes(version="0.14", post_path="news/draft-bevy-0.14/") }}
```

Comment:
Second though, I think this can be removed:

```suggestion
{{ release_notes(version="0.14") }}
```

And in `release_notes.md` use `page.colocated_path`:

```
replace(from='POST_PATH', to=page.colocated_path)
```